addon/styles/addon.scss

```diff
@@ -4,64 +4,65 @@
   position: relative;
   display: flex;
   flex-direction: row;
+
   .file-select-overlay {
     position: absolute;
+    top: 3px;
+    left: 2px;
     width: 100%;
-    height: 100%;
+    height: 35px;
     display: flex;
     flex-direction: row;
     opacity: 0;
+
+    .text {
+      display: block;
+      padding: 35px 0 0 0;
+      cursor: text;
+      width: 100%;
+    }
+
+    .button {
+      display: block;
+      padding: 35px 0 0 0;
+      cursor: pointer;
+      width: 70px;
+    }
   }
-  &.full {
+
+  .background {
+    height: 37px;
```

Comment:
35px for the content and a 2px border?  Can you check with UX to see if they want the content at 33px (or border-box) so that the final size isn't off the 5px increment?